modules/experimental-layers/src/gpu-screen-grid-layer/gpu-screen-grid-layer-vertex.glsl.js

```diff
@@ -37,16 +38,49 @@ uniform AggregationData
   vec4 maxCount;
 } aggregationData;
 
+uniform vec4 colorRange[RANGE_COUNT];
+uniform vec2 colorDomain;
+uniform bool shouldUseMinMax;
+
 out vec4 vColor;
 
+vec4 quantizeScale(vec2 domain, vec4 range[RANGE_COUNT], float value) {
```

Comment:
Thought about making this into a shader module?

Reply:
Yeah, indeed, once I we have more such methods, I will create an issue, done #1947